File: src/main/java/org/robotykhl/minigamesVoting/ConfigTab.java
package org.robotykhl.minigamesVoting;

import java.util.ArrayList;
import java.util.List;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.command.TabCompleter;

public class ConfigTab implements TabCompleter {

  MinigamesVoting main;

  public ConfigTab(MinigamesVoting main) {
    this.main = main;
  }

  @Override
  public List<String> onTabComplete(CommandSender sender, Command cmd, String s, String[] args) {
    List<String> results = new ArrayList<>();

    if (args.length == 1) {
      results.add("add");
      results.add("delete");
    } else if (args.length == 2) {
      if (args[0].equals("add")) {
        results.add("Type What To Add");
      } else if (args[0].equals("delete")) {
        for (String current : main.getConfig().getStringList("Minigames")) {
          results.add(current);
        }
      }
    }

    return results;
  }
}

File: src/main/kotlin/io/robotykhl/minigames/ConfigTab.kt
package io.robotykhl.minigames

import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.command.TabCompleter

class ConfigTab(private val main: Minigames) : TabCompleter {

    override fun onTabComplete(
        sender: CommandSender,
        cmd: Command,
        s: String,
        args: Array<String>
    ): List<String> {
        val results = mutableListOf<String>()
        if (args.size == 1) {
            results.add("add")
            results.add("delete")
            results.add("title")
            results.add("defaulttime")
        } else if (args.size == 2) {
            when (args[0]) {
                "add" -> results.add("Type What To Add")
                "delete" -> results.addAll(main.config.getStringList("Minigames"))
                "title" -> results.add("Type new title")
                "defaulttime" -> results.add("Type new default time")
            }
        }
        return results
    }
}